### DISCUSSION TITLE
How can use Aggregations(Avg, Sum, Count, Group by, ...) in Version 11?

### DISCUSSION THREAD

**Discussion by Vadoudi** [ChilliCream/hotchocolate#3946] [Q&A]
Hello,

I want to use aggregation(sum, count, avg, ...) by using `EF 5`, `HotChocolate.AspNetCore`, and `HotChocolate.Data.EntityFramework`.
**How can I do that?**

This is my `GraphQL` configuration:
![image](https://user-images.githubusercontent.com/47804206/125405370-3ec6cb00-e3ea-11eb-894f-10baf997e286.png)

These are libraries that I used in my project:
![image](https://user-images.githubusercontent.com/47804206/125405108-f14a5e00-e3e9-11eb-930c-6c5981836b48.png)

Best regards

**Comment by PascalSenn** [comment]
We already have a discussion about this feature:
https://github.com/ChilliCream/hotchocolate/discussions/2963

**Reply by Vadoudi** [reply]
@PascalSenn Thank you for your answer, I saw that discussion but how can I implement the backend for a group by?
